roadmap/parachains.md

```diff
@@ -0,0 +1,184 @@
+# Parachains Roadmap
+This is a roadmap for the core technology underlying Parachains - what protocols, APIs, and code paths need to be in place to fully instantiate a self-sufficient and secure parachain. We don't attempt to cover anything on what APIs a parachain toolkit might expose in order to make use of parachain features - only how those features are implemented and the low-level APIs that they expose to the validation function, if any.
```

Comment:
Probably having an at most 120 characters width requirement for `.md` files would prove useful.